mungegithub/mungers/milestone-maintainer.go

```diff
@@ -279,6 +281,130 @@ func durationGreaterThanZero(name string, value time.Duration) error {
 	return nil
 }
 
+func dayPhrase(days int) string {
+	dayString := "days"
+	if days == 1 || days == -1 {
+		dayString = "day"
+	}
+	return fmt.Sprintf("%d %s", days, dayString)
+}
+
+func durationToMinDays(duration time.Duration) string {
+	days := int(math.Floor(duration.Hours() / 24))
+	return dayPhrase(days)
+}
+
+func durationToMaxDays(duration time.Duration) string {
+	days := int(math.Floor(duration.Hours() / 24))
```

Comment:
s/Floor/Ceil/

Reply:
@marun are you okay with me making that change?